src/main/java/io/cdap/plugin/gcp/bigquery/sink/AbstractBigQuerySinkConfig.java

```diff
@@ -70,6 +72,11 @@
   @Description("Type of write operation to perform. This can be set to Insert, Update or Upsert.")
   protected String operation;
 
+  @Name(NAME_WRITE_DISPOSITION)
+  @Macro
+  @Description("Type of write disposition to use. This can be set to Empty, Truncate, or Append.")
+  protected String writeDisposition;
```

Comment:
this should become a Boolean for whether to truncate the table or not, which means 'Empty' is no longer supported. The use cases we have are all truncate, we can add another "Fail if not Empty" property in the future if anybody asks for it.

Reply:
Fixed!